Dockerfile.build

```diff
@@ -43,12 +43,17 @@ ARG LLVM_REPO_NAME
 RUN wget --quiet -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
 RUN add-apt-repository "$LLVM_REPO_NAME"
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends clang-format clang
+    apt-get install -y --no-install-recommends clang-format-11 clang-11
+# Versioned LLVM releases don't come with the meta packages for the
+# clang -> clang-12 symlinks. We create them manually here.
+RUN ln -s /usr/bin/clang-11 /usr/bin/clang
+RUN ln -s /usr/bin/clang++-11 /usr/bin/clang++
+RUN ln -s /usr/bin/clang-format-11 /usr/bin/clang-format
```

Comment:
The official meta package links these directly: `/usr/bin/clang -> ../lib/llvm-13/bin/clang`.
We should do that as well.